Content.Shared/Physics/CollisionGroup.cs

```diff
@@ -56,9 +56,9 @@ public enum CollisionGroup
     TableLayer = MidImpassable,
 
     // Tabletop machines, windoors, firelocks
-    TabletopMachineMask = Impassable | HighImpassable,
+    TabletopMachineMask = Impassable,
```

Comment:
Leave it on the mask so that it can't go through closed airlocks. Just removing it from the layer should allow items to be placed on top.